backend/controller/user.go

```diff
@@ -31,11 +37,17 @@ func CreateUser(w http.ResponseWriter, r *http.Request) {
 	err := json.NewDecoder(r.Body).Decode(&user)
 	// essentially reset if the user inputted anything to common as it should not be editable
 	user.Common = models.Common{}
-	helpers.CreateNewRecord(w, user, err)
+	err2, user := helpers.CreateNewRecord(w, user, err)
```

Comment:
i typically just redefine `err, user =` i don't know if that's less confusing / changes readability in any way but that's what ive been doing.